### PR TITLE
Update documentation on VideoTileState class

### PR DESCRIPTION
**Pull request by XHatan**
**Issue #:** 

**Description of changes:**

**Testing**

1. Have you successfully run `npm run build:release` locally?
2. How did you test these changes?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/videotile/VideoTileState.ts [hunk 1]
```diff
@@ -5,25 +5,107 @@
  * [[VideoTileState]] encapsulates the state of a [[VideoTile]]
  */
 export default class VideoTileState {
+  /**
+   * The unique identifier for the [[VideoTile]] managed by [[VideoTileController]]
```
Comment (vidya-mohan) [inline src/videotile/VideoTileState.ts:9]: Perhaps add - there wll be one per attendee?

File: src/videotile/VideoTileState.ts [hunk 2]
```diff
@@ -5,25 +5,107 @@
...
+   */
   active: boolean = false;
+
+  /**
+   * Indication of whether the tile has paused video stream
+   */
   paused: boolean = false;
+
+  /**
+   * Indication of whether the tile is marked with poor connection
```
Comment (vidya-mohan) [inline src/videotile/VideoTileState.ts:39]: can we qualify what is a poor connection? Is it the pongs to server has failed?

File: src/videotile/VideoTileState.ts [hunk 3]
```diff
@@ -5,25 +5,107 @@
...
+   */
   videoElementPhysicalWidthPixels: number | null = null;
+
+  /**
+   * The physical height in pixel of the HTMLVideoElement upon binding with the [[VideoTile]]
+   */
   videoElementPhysicalHeightPixels: number | null = null;
+
+  /**
+   * The unique identifier for the video stream bound with the [[VideoTile]]
```
Comment (vidya-mohan) [inline src/videotile/VideoTileState.ts:106]: Can we say its the WebRTc MediaStreamId?

File: src/videotile/VideoTileState.ts [hunk 4]
```diff
@@ -5,25 +5,107 @@
...
+   */
   boundVideoElement: HTMLVideoElement | null = null;
+
+  /**
+   * The nameplate for the [[VideoTile]]. SDK users should use boundExternalUserId for user id
+   */
   nameplate: string | null = null;
+
+  /**
+   * The intrinsic width of the video stream upon binding with the [[VideoTile]].
```
Comment (vidya-mohan) [inline src/videotile/VideoTileState.ts:69]: can we add a link to https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html#how-can-i-create-a-video-tile-layout-for-my-application on how to do it?

File: src/videotile/VideoTileState.ts (hunk 1 above)
Comment (XHatan) [reply src/videotile/VideoTileState.ts:9]: sgtm

File: src/videotile/VideoTileState.ts (hunk 2 above)
Comment (XHatan) [reply src/videotile/VideoTileState.ts:39]: good suggestion, going to add link on how this is determined

File: src/videotile/VideoTileState.ts (hunk 3 above)
Comment (XHatan) [reply src/videotile/VideoTileState.ts:106]: this is streamId defined in signaling protocol, going to clarify

File: guides/06_FAQs.md [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 ### What is the Amazon Chime SDK?
 
-The [Amazon Chime SDK](https://aws.amazon.com/chime/chime-sdk) is a set of real-time communications components that developers can use to quickly add audio calling, video calling, and screen sharing capabilities to their own web or mobile applications. Developers can leverage the same communication infrastructure and services that power Amazon Chime, an online meetings service from AWS, and deliver engaging experiences in their applications. 
+The [Amazon Chime SDK](https://aws.amazon.com/chime/chime-sdk) is a set of real-time communications components that developers can use to quickly add audio calling, video calling, and screen sharing capabilities to their own web or mobile applications. Developers can leverage the same communication infrastructure and services that power Amazon Chime, an online meetings service from AWS, and deliver engaging experiences in their applications.
```
Comment (XHatan) [inline guides/06_FAQs.md:7]: no change, trailing space is removed.

File: guides/06_FAQs.md [hunk 6]
```diff
@@ -72,7 +72,7 @@ AWS accounts have a soft limit of [250 concurrent meetings](https://docs.aws.ama
 
 ### How many attendees can join an Amazon Chime SDK meeting? Can this limit be raised?
 
-Amazon Chime SDK limits are defined [here](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-limits). The service supports up to 250 attendees and up to 16 video participants in a meeting (video limit is enforced separately). An attendee is considered active unless it has been explicitly removed using DeleteAttendee API call. Attendee limits cannot be changed. 
+Amazon Chime SDK limits are defined [here](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-limits). The service supports up to 250 attendees and up to 16 video participants in a meeting (video limit is enforced separately). An attendee is considered active unless it has been explicitly removed using DeleteAttendee API call. Attendee limits cannot be changed.
```
Comment (XHatan) [inline guides/06_FAQs.md:75]: no change, trailing space is removed.

File: guides/06_FAQs.md [hunk 7]
```diff
@@ -184,7 +184,7 @@ This is a known issue on iPhone that seems to happen in iPhone X and XR versions
 
 ### **I notice that if I turn on camera and put the browsers in background, others in the meeting will see black tile in Safari in iOS and frozen tile in Chrome in Android, is this a known issue?**
 
-This is the default behavior that is specific to each browser. In Android, the video stream is muted when in background and thus, the video will show the last frame. 
+This is the default behavior that is specific to each browser. In Android, the video stream is muted when in background and thus, the video will show the last frame.
```
Comment (XHatan) [inline guides/06_FAQs.md:187]: no change, trailing space is removed

File: guides/06_FAQs.md [hunk 8]
```diff
@@ -193,7 +193,7 @@ This seems to be a [bug](https://bugs.chromium.org/p/webrtc/issues/detail?id=116
...
+This error indicates that the device you are using does not support hardware acceleration decoding. However this does not impact the ability of this user to participate in Amazon Chime SDK meetings as the device can render and transmit VP8 streams to other parties in the call. Specifically for Chrome, you will need to enable Unified Plan support by setting this [flag](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html#enableunifiedplanforchromiumbasedbrowsers) to true or enable [Simulcast](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html#enablesimulcastforunifiedplanchromiumbasedbrowsers).
```
Comment (XHatan) [inline guides/06_FAQs.md:196]: no change

**Pull request closed without merging**